sandbox/functions-recipes/includes/durable-logging.md

```diff
@@ -0,0 +1,33 @@
+## Logging in orchestrator functions
+An orchestrator function creates checkpoints as each of its composite activity functions complete. After each call to `CallActivityAsync` on the `DurableOrchestrationContext` instance completes, the orchestrator function will automatically replay to rebuild their in-memory state.
+
+To prevent logging statements from duplicating, use the `IsReplaying` property on the `DurableOrchestrationContext`.
+
+```csharp
+[FunctionName("PlaceOrder")]
+public static async Task<InvoiceData> OrderOrchestration(
+    [OrchestrationTrigger] DurableOrchestrationContext context,
+    TraceWriter log)
```

Comment:
FYI, we're trying to deprecate `TraceWriter` and having all of our samples use `ILogger` instead.